example/lib/domino_sample.dart

```diff
@@ -33,35 +74,36 @@ class DominoBrick extends BodyComponent {
 
   @override
   Body createBody() {
-    FixtureDef fd = FixtureDef();
-    PolygonShape sd = PolygonShape();
-    sd.setAsBoxXY(0.125, 2.0);
-    fd.shape = sd;
-    fd.density = 25.0;
+    final shape = PolygonShape()..setAsBoxXY(0.125, 2.0);
+    FixtureDef fixtureDef = FixtureDef(shape)
+      ..density = 25.0
+      ..friction = .5;
 
-    BodyDef bd = BodyDef();
-    bd.type = BodyType.DYNAMIC;
-    bd.position = position;
+    BodyDef bodyDef = BodyDef()
+      ..type = BodyType.dynamic
+      ..position = position;
 
-    fd.friction = .5;
-    return world.createBody(bd)..createFixture(fd);
+    return world.createBody(bodyDef)..createFixture(fixtureDef);
   }
 }
 
 class DominoSample extends Forge2DGame with TapDetector {
   @override
   bool debugMode = true;
 
+  late Image _pizzaImage;
+
   DominoSample()
       : super(
-          scale: 8.0,
+          scale: 20.0,
           gravity: Vector2(0, -10.0),
         );
 
   @override
   Future<void> onLoad() async {
-    final boundaries = createBoundaries(viewport);
+    final boundaries = createBoundaries(viewport as Forge2DViewport);
```

Comment:
I wonder why this casting is now necessary

Reply:
Before it was using it's own viewport, now it is using a subclass of the Flame viewport